src/Symfony/Bundle/FrameworkBundle/Resources/public/css/exception.css

```diff
@@ -56,14 +56,17 @@
 }
 .sf-reset .text-exception .open-quote,
 .sf-reset .text-exception .close-quote {
+    font-family: Arial, Helvetica, sans-serif;
```

Comment:
@pborreli arial is nice :)
![capture du 2016-08-29 15-59-54](https://cloud.githubusercontent.com/assets/243674/18053845/aa1d2e26-6e01-11e6-9d74-5cd168c6f051.png)